scripts/init-submodules-no-riscv-tools.sh

```diff
@@ -6,6 +6,18 @@ set -o pipefail
 
 RDIR=$(git rev-parse --show-toplevel)
 
+NO_FIRESIM=false
+
+while test $# -gt 0
```

Comment:
I prefer to use the standard `getopts` built-in for options parsing.  It's more flexible if we have more flags in the future, since it accounts for merging (`-ab` is equivalent to `-a -b`) and argument separation (`-f<arg>` is equivalent to `-f <arg>`).

It doesn't natively support GNU-style long options, but this can be worked around:
```sh
_usage() {
  echo "usage ${0} [--no-firesim]" >&2
  exit 1
}

while getopts '-:' opt ; do
  case ${opt} in
  -)
    case ${OPTARG} in
    no-firesim) NO_FIRESIM=true ;;
    *) echo "invalid option: --${OPTARG}" >&2 ; _usage ;;
    esac ;;
  *) echo "invalid option: -${opt}" >&2 ; _usage ;;
  esac
done
shift $((OPTIND - 1))
```

Reply:
switched